pyproject.toml

```diff
@@ -84,5 +84,10 @@ doc = [
 [project.scripts]
 ihatemoney = "ihatemoney.manage:cli"
 
-[tool.setuptools]
-packages = ["ihatemoney"]
+[tool.hatch.build.hooks.custom]
+dependencies = [
+    "Babel>=2.6.0"
+]
+
+[tool.hatch.build]
```

Comment:
So, the [hatch_build.py](https://github.com/spiral-project/ihatemoney/pull/1258/files#diff-d56d1b58c0ca28bdddd10c2cee2ab3cb4312766ea5de51f210dbc15a0fea42fa) file is called implicitly by hatch ?

Reply:
Yes.